### ISSUE TITLE
make library file more portable

### ISSUE THREAD

**Issue by waiwaa** [sonofeft/RocketCEA#4]
Prebuilt pyd/so library file  has dependency on libgfortran. That makes every user to install complete compiling toolchains, which increases cost of using this package. May be you can **compile the module with f2py in a "static" way** to get a more portable  library.

- run f2py command with --build-dir [build_dir] option, thus you can keep intermediate files.
- repeat the linking command (get it from stdout of the previous f2py command) , adding " -static " linker option.

I test this in Windows, and it works without libgfortran dependency. 

Finally, thanks for the great work of this project,.

**Comment by sonofeft** [comment]
Thanks for your interest in RockeCEA, and for your suggestion.

I started out trying to prebuild pyd/so files for all platforms but it turned out to be very difficult to support.  
My original matrix was 
(Windows, Linux, Mac) * (python 2.7, 3.6 and 3.7) * (32 bit and 64 bit)
(Also in those 18 cases there are variations in operating systems that should have been considered, and, perhaps, some other operating systems should have been included.)

I ended up trying to trouble-shoot pyd/so problems on systems with which I had no experience or access.

Since there is no monetary cost in the open source FORTRAN compiler required to install RocketCEA, I refactored RocketCEA to do the FORTRAN compile on the user's machine.

Also, I think it's a benefit to most RocketCEA users to be familiar with a free, open-source FORTRAN compiler. In my corporate career I've had trouble maintaining code bases with licensed, commercial FORTRAN compilers.

On balance, I think this approach will work out better for most users and certainly for me ;-)

Again, thanks for your suggestion. My original instinct was the same approach you are suggesting.
